R/pkg/R/DataFrame.R

```diff
@@ -936,7 +936,9 @@ setMethod("unique",
 
 #' Sample
 #'
-#' Return a sampled subset of this SparkDataFrame using a random seed.
+#' Return a sampled subset of this SparkDataFrame using a random seed. 
+#' Note that this is not guaranteed to provide exactly the fraction specified
```

Comment:
@anabranch it is already probably fine (I hope this does not sound like nitpicking) but maybe we would better just match this up as below (there are some examples for this in https://github.com/anabranch/spark/blob/b4f2611214e00e11040d78fa33b8772588897264/R/pkg/R/functions.R#L2299-L2300 and https://github.com/anabranch/spark/blob/b4f2611214e00e11040d78fa33b8772588897264/R/pkg/R/mllib.R#L606-L607):

```
#' Return a sampled subset of this SparkDataFrame using a random seed. 
#' 
#' Note: that this is not guaranteed to provide exactly the fraction specified
```

I understand `Note:` and `NOTE:` are mixed across the documentation (including annotations and indentation). Let me try to deal with this across all documentation in a separate PR.